.github/workflows/ci.yml

```diff
@@ -116,3 +116,23 @@ jobs:
           aws-access-key-id: ${{secrets.AWS_ACCESS_KEY_ID}}
           aws-secret-access-key: ${{secrets.AWS_SECRET_ACCESS_KEY}}
           aws-session-token: ${{secrets.AWS_SESSION_TOKEN}}
+  reviewers:
+    name: Add Reviewers
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    needs:
+      - lint
+      - licenses
+      - build
+      - test
+    permissions:
+      contents: read
+      pull-requests: write
+    if: failure() && contains(github.triggering_actor, '[bot]')
```

Comment:
Hard to test but this should auto add use if any of it's needs fails and the triggering actor is a `bot`. We'll see if it works if this ever comes up.